src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,128 @@
+        System.out.println("Hello from\n" + logo);
+    }
+
+    public static void showBye() {
+        System.out.println("----------------------------------");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("----------------------------------");
+    }
+
+    /**
+     * Responds to the user by printing a response, based on the chat sent by user
+     * @param chat User input
+     * @param ls The TaskList in Duke
+     */
+    public static void respond(String chat, TaskList ls) {
+        String keyword = chat.split(" ")[0];
+        switch(keyword) {
+        case("list"): {
+            ArrayList<Task> list = ls.getList();
+            String response = "";
+            for (int i = 0; i < list.size(); i++) {
+                Task task = list.get(i);
+                response += Integer.toString(i + 1) + ". ";
+                response += task.toString();
+                response += "\n";
+            }
+            Ui.print(response);
+            break;
+        }
+        case("mark"): {
+            Task task = ls.mark(chat.substring(5));
+            String response = "Nice! I've marked this task as done:\n";
+            response += task.toString();
+            Ui.print(response);
+            break;
+        }
+        case("unmark"): {
+            Task task = ls.unmark(chat.substring(7));
+            String response = "OK, I've marked this task as not done yet:\n";
+            response += task.toString();
+            Ui.print(response);
+            break;
+        }
+        case("todo"): {
+            try {
+                Task task = ls.addToDo(chat);
+                String response = "Got it. I've added this task:\n";
+                response += task.toString() + "\n";
+                response += "Now you have " + ls.getSize() + " tasks in the list.";
+                Ui.print(response);
+            } catch (DukeException e) {
+                Ui.print(e.toString());
+            }
+            break;
+        }
+        case("deadline"): {
+            try {
+                Task task = ls.addDeadline(chat);
+                String response = "Got it. I've added this task:\n";
+                response += task.toString() + "\n";
+                response += "Now you have " + ls.getSize() + " tasks in the list.";
+                Ui.print(response);
+            } catch (DukeException e) {
+                Ui.print(e.toString());
+            }
+            break;
+        }
+        case("event"): {
+            try {
+                Task task = ls.addEvent(chat);
+                String response = "Got it. I've added this task:\n";
+                response += task.toString() + "\n";
+                response += "Now you have " + ls.getSize() + " tasks in the list.";
+                Ui.print(response);
+            } catch (DukeException e) {
+                Ui.print(e.toString());
+            }
+            break;
+        }
```

Comment:
Similarly, should these repeated lines be refactored?

```suggestion
        case("todo"):
        case("deadline"):
        case("event"): {
            respondTask(keyword, chat, ls);
            break;
        }
```

with the additional method:

```java
private static void respondTask(String keyword, String chat, TaskList ls) {
    try {
        Task task;
        switch (keyword) {
        case "todo":
            ls.addToDo(chat);
            break;
        case "deadline":
            ls.addDeadline(chat);
            break;
        case "event":
            ls.addEvent(chat);
            break;
        default:
            return;
        }
        String response = "Got it. I've added this task:\n";
        response += task.toString() + "\n";
        response += "Now you have " + ls.getSize() + " tasks in the list.";
        Ui.print(response);
    } catch (DukeException e) [
        Ui.print(e.toString());
    }
}
```